flexport/latitude
# SearchableMultiselectFilter no longer has a shadow on it's dropdown list

## Description:

This PR removes the drop-shadow on the SearchableMultiselectFilter's dropdown list.

## Manual Test Plan:
Go to storybook
open the Filters story
see that the SearchableMultiselectFilter no longer has a shadow on the dropdown list. 

## Screenshots:

Before:
![image](https://user-images.githubusercontent.com/5897853/71849373-be057100-3086-11ea-81bb-727dddcf6338.png)

After:
![image](https://user-images.githubusercontent.com/5897853/71849391-c8276f80-3086-11ea-89e4-a415af634731.png)
